<ui version="4.0" >
 <class>ProfileDialogClass</class>
 <widget class="QDialog" name="ProfileDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Change profile</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="grbProfiles" >
     <property name="title" >
      <string>Open profile</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="lblProfile" >
          <property name="text" >
           <string>Profile:</string>
          </property>
          <property name="buddy" >
           <cstring>cmbProfiles</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbProfiles" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QListWidget" name="lstProfiles" >
          <property name="selectionBehavior" >
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QPushButton" name="pbtNew" >
            <property name="text" >
             <string>New</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbtRename" >
            <property name="text" >
             <string>Rename</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbtDelete" >
            <property name="text" >
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lblLang" >
       <property name="text" >
        <string>Language:</string>
       </property>
       <property name="buddy" >
        <cstring>cmbLang</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbLang" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btbButtons" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>btbButtons</sender>
   <signal>rejected()</signal>
   <receiver>ProfileDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>361</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel" >
     <x>318</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
